store/src/java/com/zimbra/cs/account/ZAttrAccount.java

```diff
@@ -21104,6 +21104,83 @@ public Map<String,Object> unsetFeatureSharedFolderMobileSyncEnabled(Map<String,O
         return attrs;
     }
 
+    /**
```

Comment:
Do not manually edit this.
You need to follow these steps:
To add new ldap attribute
```
    1. add new attribute in zm-mailbox/store/conf/attrs/zimbra-attrs.xml
    2. cd store
    3. ant generate-getters
```